experimental/regression_suite/ireers/fixtures.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+
+from typing import Dict, Sequence, Union
+from pathlib import Path
+import subprocess
+import time
+
+from .artifacts import (
+    Artifact,
+    FetchedArtifact,
+    ProducedArtifact,
+)
+
+
+IREE_COMPILE_QOL_FLAGS = [
+    "--mlir-timing",
+    "--mlir-timing-display=tree",
+    "--iree-consteval-jit-debug",
+]
+
+
+def fetch_source_fixture(url: str, *, group: str):
+    @pytest.fixture
+    def fetcher() -> FetchedArtifact:
+        art = FetchedArtifact(url=url, group=group)
+        art.start()
+        return art
+
+    return fetcher
+
+
+def iree_compile(source: Artifact, compiled_variant: str, flags: Sequence[str]):
+    name = Path(source.name).with_suffix(f".{compiled_variant}.vmfb")
+
+    def callback(vmfb_artifact: ProducedArtifact):
+        sep = "\n  "
+        print("**************************************************************")
+        print(f"Compiling {source} -> {vmfb_artifact} with flags:")
```

Comment:
How about print the full command including `iree-compile -o {vmfb_artifact.path} {source.path}` so users can directly copy and paste to compile by themselves?

Reply:
I'm going to revamp this all very shortly, make logging very verbose/persistent/etc. Just getting bootstrapped...